core/src/test/java/de/evoila/cf/broker/controller/core/ServiceInstanceBindingControllerTest.java

```diff
@@ -0,0 +1,731 @@
+package de.evoila.cf.broker.controller.core;
```

Comment:
This test class is HUGE. It would be a great improvement, if you found a way to split this class into several classes, so debugging would be easier.

I am thinking about creating a base class that holds the constants and such necessary base features and then split into lifecycle stages, like create, update, delete.